gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/step/map/GraphStep.java

```diff
@@ -164,13 +164,13 @@ public int hashCode() {
     }
 
     /**
-     * Attemps to close an underlying iterator if it is of type {@link CloseableIterator}. Graph providers may choose
+     * Attempts to close an underlying iterator if it is of type {@link CloseableIterator}. Graph providers may choose
      * to return this interface containing their vertices and edges if there are expensive resources that might need to
      * be released at some point.
      */
     @Override
     public void close() throws Exception {
-        if (iterator != null && iterator instanceof CloseableIterator) {
```

Comment:
`GraphStep` doesn't extend `FlatMapStep`, so it doesn't have access to `closeIterator()`, but it can call `CloseableIterator.closeIterator(iterator)`